buildenv/jenkins/Jenkinsfile_personal

```diff
@@ -11,7 +11,7 @@ pipeline {
 		choice (choices: 'openjdk8\nopenjdk8-openj9\nopenjdk9\nopenjdk9-openj9\nopenjdk10\nopenjdk10-openj9\nopenjdk10-sap', description: 'What is JVM Version?', name: 'JVM_VERSION')
 		string (defaultValue: '', description: 'Specific test project, leave blank by default for all projects to be compiled, e.g. openjdk_regression systemtest performance jck thirdparty_containers', name: 'TESTPROJECT')
 		string (defaultValue: 'runtest', description: 'Test Target to execute, default is runtest which will run all tests. You can also select sub-targets such as:\ndifferent group: openjdk, system, jck, functional, perf, etc\ndifferent level: sanity, extended\nindividual test: jdk_beans_0, etc', name: 'TARGET')
- 		choice (choices: 'linux&&x64&&test\nmac&&x64&&test\nwindows&&x64&&test\nsw.tool.docker', description: 'What is machine label?', name: 'MACHINE_LABEL')
+ 		choice (choices: 'linux&&x64&&test\nmac&&x64&&test\nwindows&&x64&&test\nsw.tool.docker\nci.project.openj9&&sw.os.linux&&hw.arch.x86\nci.project.openj9&&sw.os.linux&&hw.arch.s390\nci.project.openj9&&hw.arch.ppc&&sw.os.aix7.1', description: 'What is machine label?', name: 'MACHINE_LABEL')
```

Comment:
This will be changed to String type, which user can input whatever workable label. https://github.com/AdoptOpenJDK/openjdk-tests/pull/308

Reply:
Thanks for pointing it out. The change you mentioned above has been merged to this PR. Please review @sophia-guo